x-pack/plugins/cloud_security_posture/public/pages/findings/findings_distribution_bar.tsx

```diff
@@ -0,0 +1,159 @@
+import { css } from '@emotion/react';
+import { EuiTextColor, EuiSpacer, EuiFlexGroup, EuiFlexItem, useEuiTheme } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import numeral from '@elastic/numeral';
+
+interface Props {
+  total: number;
+  passed: number;
+  failed: number;
+  pageStart: number;
+  pageEnd: number;
+}
+
+const formatNumber = (value: number) => (value < 1000 ? value : numeral(value).format('0.0a'));
+
+export const FindingsDistributionBar = (props: Props) => (
+  <div>
+    <Counters {...props} />
+    <EuiSpacer size="s" />
+    <DistributionBar {...props} />
+  </div>
+);
+
+const Counters = ({ pageStart, pageEnd, total, failed, passed }: Props) => (
+  <EuiFlexGroup justifyContent="spaceBetween">
+    <EuiFlexItem>
+      <CurrentPageOfTotal pageStart={pageStart} pageEnd={pageEnd} total={total} />
+    </EuiFlexItem>
+    <EuiFlexItem
+      css={css`
+        align-items: flex-end;
+      `}
+    >
+      <PassedFailedCounters passed={passed} failed={failed} />
+    </EuiFlexItem>
+  </EuiFlexGroup>
+);
+
+const PassedFailedCounters = ({ passed, failed }: Pick<Props, 'passed' | 'failed'>) => {
+  const { euiTheme } = useEuiTheme();
+  return (
+    <div
+      css={css`
+        display: flex;
+      `}
+    >
+      <Counter
+        label={i18n.translate('xpack.csp.findings.distributionBar.totalPassedLabel', {
+          defaultMessage: 'Passed',
+        })}
+        color={euiTheme.colors.success}
+        value={passed}
+      />
+      <Counter
+        label={i18n.translate('xpack.csp.findings.distributionBar.totalFailedLabel', {
+          defaultMessage: 'Failed',
+        })}
+        color={euiTheme.colors.danger}
+        value={failed}
+      />
+    </div>
+  );
+};
+
+const CurrentPageOfTotal = ({
+  pageEnd,
+  pageStart,
+  total,
+}: Pick<Props, 'pageEnd' | 'pageStart' | 'total'>) => (
+  <EuiTextColor color="subdued">
+    {total > 0 && (
+      <FormattedMessage
+        id="xpack.csp.findings.distributionBar.showingPageOfTotalLabel"
+        defaultMessage="Showing {pageStart}-{pageEnd} of {total} Findings"
+        values={{
+          pageStart: <b>{pageStart}</b>,
+          pageEnd: <b>{pageEnd}</b>,
+          total: <b>{formatNumber(total)}</b>,
+        }}
+      />
+    )}
+  </EuiTextColor>
+);
+
+const DistributionBar: React.FC<Props> = ({ passed, failed }) => {
+  const { euiTheme } = useEuiTheme();
+  return (
+    <div
+      css={css`
+        display: flex;
+        align-items: center;
+        width: 100%;
+        height: 8px;
+        background: ${euiTheme.colors.subdued};
+      `}
+    >
+      <div
+        css={css`
+          flex: ${passed};
+          background: ${euiTheme.colors.success};
+          height: 100%;
+        `}
+      />
+      <div
+        css={css`
+          flex: ${failed};
+          background: ${euiTheme.colors.danger};
+          height: 100%;
+        `}
+      />
+    </div>
+  );
```

Comment:
try using `EuiFlexItem`

Reply:
it didn't work unfortunately 

we use the `flex` shorthand to assign the `flex-grow` property. this code uses the actual `passed` / `failed` number, as in - `20,000` or `13,400` for example. 


this doesn't work with the `grow` property of `EuiFlexItem`, as we get this error: 
> Prop `grow` passed to `EuiFlexItem` must be a boolean or an integer between 1 and 10, received `0.7674200765131352`

* i changed the values to be `(passed OR failed) / total)` so it ends up being `0.N` instead of a large number. 

in anyway, turns out the max possible value for all CSS `<number>` values is `2,147,483,647` which is a 32bit integer. 

all of this to say - using a plain `div` seems the most fitting, because we can't use `grow` with `EuiFlexItem`, which is what we need to scale each part of the distribution bar. also, we would need the `background` and `height` css anyway for this case.